index.html

```diff
@@ -51,7 +51,13 @@
                                 <div class="navBar-header">
                                     <h1>iPaidThat</h1>
                                 </div>
-                            </div>
+							</div>
```

Comment:
Try to be consistent with code style, indentation should be the same throughout the project